scaffolder-templates/gitlab-workflows/convert-workflow-to-template/template.yaml

```diff
@@ -0,0 +1,119 @@
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: convert-workflow-to-template
```

Comment:
IINW the name should be unique - let's add the `-gitlab` as suffix and `-github` for the other one.